massa-node/base_config/config.toml

```diff
@@ -175,6 +175,8 @@
     max_operations_propagation_time = 32000
     # time threshold after which endorsement are not propagated
     max_endorsements_propagation_time = 48000
+    # number of thread tester
+    thread_tester_count = 5
```

Comment:
maybe default thread count should be less than 5

Reply:
I find 5 be a good number